composer.json

```diff
@@ -23,15 +23,15 @@
   ],
   "require": {
     "php": ">=7.1",
-    "fzaninotto/faker": "^1.4",
-    "intervention/image": "^2.4"
+    "fakerphp/faker": "^1.13.0",
```

Comment:
I think you are right! But change to `^1.13` please

But then I also should republish my package with new subversion...